packages/jellyfish-transaction-builder/src/txn/txn_builder_loans.ts

```diff
@@ -33,4 +35,19 @@ export class TxnBuilderLoans extends P2WPKHTxnBuilder {
       changeScript
     )
   }
+
+  /**
+   * Destroy a loan scheme. Currently requires Foundation Authorization.
+   *
+   * @param {DestroyLoanScheme} destroyLoanScheme txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @returns {Promise<TransactionSegWit>}
+   */
+  async destroyLoanScheme (destroyLoanScheme: DestroyLoanScheme, changeScript: Script): Promise<TransactionSegWit> {
+    destroyLoanScheme.height = destroyLoanScheme.height ?? new BigNumber(0x00000000)
```

Comment:
height is not optional

Reply:
@surangap

Reply:
https://github.com/DeFiCh/jellyfish/pull/594/files#r691771469

Reply:
resolved by b55c2acf89e3595bc157b785a7b9a3ae71749c0b